suites/fs/recovery/tasks/strays.yaml

```diff
@@ -0,0 +1,5 @@
+
+tasks:
+    - cephfs_test_runner:
+        modules:
+            - tasks.cephfs.test_strays
```

Comment:
The indentation on these task descriptions is varying pretty widely — does it all work? :/ (It looks like maybe you switched to 4-space indentation, but I think all the existing yaml fragments are 2-spaced.)